nucliadb_vectors2/src/service/writer.rs

```diff
@@ -59,6 +59,19 @@ impl Debug for VectorWriterService {
     }
 }
 
+fn record_merge_metrics(source: MergeSource, data: &crate::data_point_provider::writer::MergeMetrics) {
+    if data.merged == 0 {
```

Comment:
Maybe knowing how often this happens could help us in some way? To tweak frequencies maybe?